packages/core2/src/lib/prismicrc.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from "node:path";
+import * as t from "io-ts";
+import * as rc9 from "rc9";
+
+import { decode } from "./decode";
+
+const PRISMICRC = ".prismicrc";
```

Comment:
Yes, I think people should be able to set a global preference (as an individual, I don't want to be tracked) in the home folder, while also being able to set a local preference (as an organization, I don't want this project to be tracked) I'll update it in that fashion.

- I know we have a `~/.prismic` file there already and those should eventually be merged into one.
- I don't know about the syntax, on one hand, we should likely stick with the JSON syntax for compatibility, on another hand I don't like that the file name does not git any hint about its syntax.
- I feel like the `rc` suffix/syntax is a bit more common than non-`rc` suffixed/syntax files

To be continued~

Reply:
`~/.prismic` is a state persistence file, while `~/.prismicrc` is a user configuration file. They probably should be separate files since `~/.prismic` will be updated programmatically.

Unfortunately, their locations and names are confusing since they are so similar.

Alternatives:

- Use the `~/.config/prismic` directory, which follows the standard `XDG_CONFIG_HOME` location on Unix-like systems. I'm not sure if Windows has a similar convention.
- Name `~/.prismic` something different, like `.prismic-session`